CONTRIBUTING.md

```diff
@@ -0,0 +1,104 @@
+# Contributing to Headset
+
+Welcome to the Headset community!
+
+## Community
+
+We have two main channels of communication:
+- our Slack workspace: https://tinyurl.com/y7m8y5x4
+- issues on GitHub
+
+If you have any questions on where to start and/or how to proceed, let us know!
+
+## Ways to contribute
+
+We welcome many ways of contributing to Headset:
+- finding bugs and [reporting issues](https://github.com/headsetapp/headset-electron/issues)
+- picking and working on an existing [issue](https://github.com/headsetapp/headset-electron/labels/help-wanted) marked as 
+  "help wanted"
+- proposing and implementing improvements to Headset
+
+## Setting up the environment
+
+1. Fork the repository on GitHub
+
+2. Clone your fork of the repository
+
+  ```
+  git clone https://github.com/<your-username>/headset-electron.git
+  ```
+
+3. Installation is OS-dependent, so choose the appropriate OS directory:
+  - `windows` for Windows
+  - `darwin` for OSX
+  - `linux` for Linux
+
+    To install all the required dependencies run:
```

Comment:
I think it should be mentioned that this should be run from the root directory. Maybe start by saying:

`Installation is OS-dependent. To install all the required dependencies run from the main Headset directory:...`

And after the command is shown, mention the list of OSs that people can choose

Reply:
at this point I think users will be in the root so the next line `npm i && cd <your-os> && npm i` implying that

Reply:
I'll leave it as it is :)